### PR TITLE
[PB-960]: fix/hide pending invitations button

### PR DESCRIPTION
**Pull request by rafijv**
- Display manage access action for files in shared view
- Hide invitation button if no invitations
- Fix reload shared view after manage access

### CODE REVIEW

File: src/app/share/views/SharedLinksView/SharedView.tsx [hunk 1]
```diff
@@ -121,7 +121,14 @@ export default function SharedView(): JSX.Element {
 
   useEffect(() => {
     if (!currentFolderId && !isShareDialogOpen && !folderUUID) {
-      setTimeout(fetchRootFolders, 200);
+      setTimeout(() => {
+        setPage[0];
```
Comment (CandelR) [inline src/app/share/views/SharedLinksView/SharedView.tsx]: The function is not being executed here. `setPage[0]` -> `setPage(0);`
In `onShowInvitationsModalClose` is the same error, change it in this PR please.

File: src/app/share/views/SharedLinksView/SharedView.tsx [hunk 2]
```diff
@@ -121,7 +121,14 @@ export default function SharedView(): JSX.Element {
...
+        setShareItems([]);
+        setHasMoreRootFolders(true);
+        setHasMoreFolders(true);
+        setHasMoreItems(true);
+        fetchRootFolders();
```
Comment (CandelR) [inline src/app/share/views/SharedLinksView/SharedView.tsx]: If all of these it is for reset the Shared view state, add it to a function and reuse it in all places that is need it.

**Review by CandelR — CHANGES_REQUESTED** [review]

File: src/app/share/views/SharedLinksView/SharedView.tsx (hunk 1 above)
Comment (rafijv) [reply src/app/share/views/SharedLinksView/SharedView.tsx]: Done

File: src/app/share/views/SharedLinksView/SharedView.tsx (hunk 2 above)
Comment (rafijv) [reply src/app/share/views/SharedLinksView/SharedView.tsx]: Done

**Review by CandelR — APPROVED** [review]

**Pull request merged**